test/fixture/native-app1/package.json

```diff
@@ -18,7 +18,7 @@
     "ffi-napi": "2.4.5"
   },
   "dependencies": {
-    "@newrelic/native-metrics": "5.3.0",
+    "@newrelic/native-metrics": "7.0.2",
```

Comment:
is this related to the `colors` change, or related to the current CI windows failures?

Reply:
I was trying to fix the CI Windows failures :disappointed: